docs/source/examples/monitoring.rst

```diff
@@ -0,0 +1,56 @@
+Monitoring TorchJD
+==================
+
+
```

Comment:
We need a description of this code. I am thinking of something like (need to add links to the torch docs?):
```
The class :doc:`Aggregator<docs/aggregation/bases>` is a subclass of `nn.Modules` from `torch`. This allows registering hooks, which can be use to monitor some information about aggregations. The following code example demonstrates registering a hook to compute the cosine similarity between the aggregation performed by `Upgrad` and the average of the gradients. and another to compute the weights of the weighting of UPGrad 
```